src/main/webapp/view/lecture-view.css

```diff
@@ -115,6 +115,19 @@ iframe {
   width: 100%;
 }
 
+/* Comment Indicator */
```

Comment:
I think I already saw this code. Does it need to be synced?

Reply:
sorry about that, just synced it. Thanks!